SimG4Core/CustomPhysics/interface/CMSSIMPInelasticProcess.h

```diff
@@ -0,0 +1,167 @@
+#ifndef SimG4Core_CustomPhysics_CMSSIMPInelasticProcess_H
+#define SimG4Core_CustomPhysics_CMSSIMPInelasticProcess_H
+
+#include "G4HadronicProcess.hh"
+
+class G4ParticleDefinition;
+
+class CMSSIMPInelasticProcess : public G4HadronicProcess {
```

Comment:
@fabiocos , most of virtual methods must be removed - they are implemented inside G4HadronicProcess. In different Geant4 releases they may be different.

Reply:
@fabiocos , The code was working in 10.0 where we run in single threading mode, in 10.6 many things were changed except interfaces, so the code is well compiling but there may be problems, which I cannot catch promptly.